README.md

```diff
@@ -33,89 +33,92 @@ System.Web.Bem - БЭМ-инфрастурктура для ASP.NET MVC.
+- reference for .NET library named System.Web.Bem is added (this library allows to use BEMHTML templates on the server side).
 
-### Структура проекта
+### Project structure
 
 ```
-<Project root>                  // корневая папка проекта
-├─ Bem                          // папка БЭМ-проекта
+<Project root>
+├─ Bem                          // BEM-files folder
 │  ├─ .enb
-│  │  └─ make.js                // конфиг сборщика enb
-│  ├─ desktop.blocks            // уровень переопределения, внутри находятся блоки
+│  │  └─ make.js                // enb configuration file
+│  ├─ desktop.blocks            // redefinition level (blocks are located inside)
 │  │  ├─ block-1 
 │  │  ├─ block-2 
 │  │  │  ... 
 │  │  └─ block-n 
-│  │     ├─ block-n.bemhtml.js  // реализация блока block-n в технологии bemhtml.js (шаблоны)
-│  │     ├─ block-n.css         // реализация блока block-n в технологии css 
+│  │     ├─ block-n.bemhtml.js  // implementation of 'block-n' in 'bemhtml.js' technology (templates)
+│  │     ├─ block-n.css         // implementation of 'block-n' in 'css' technology
 │  │     │  ...                 // ...
 │  │     └─ block-n.js
-│  ├─ desktop.bundles           // папка с бандлами проекта
+│  ├─ desktop.bundles           // folder for the bundles
 │  │  ├─ bundle-1 
 │  │  ├─ bundle-2 
 │  │  │  ... 
 │  │  └─ bundle-n 
-│  │     └─ bundle-n.bemdecl.js // декларация бандла bundle-n 
-│  └─ levels.js                 // список уровней переопределения
+│  │     └─ bundle-n.bemdecl.js // declaration of the bundle named 'bundle-n'
+│  └─ levels.js                 // list of redefinition levels
 │  ...
-├─ Controllers                  // Controllers, Models, Views - стандартные папки ASP.NET MVC
+├─ Controllers                  // Controllers, Models, Views - usual ASP.NET MVC folders
 ├─ Models
 ├─ Views
 │  ...
-├─ package.json                 // конфиг npm
-└─ Web.config                   // конфиг вашего приложения
+├─ package.json                 // npm configuration file
+└─ Web.config                   // configuration file of your application
 ```
 
-### Сборка
-**Внимание! Для сборки проекта необходимо, чтобы на компьютере был установлен [node.js](https://nodejs.org/en/). Для работы приложения на сервере устанавливать node.js нет необохдимости.**
+### Build the bundles
+**Note! Node.js is required for project building. Node.js on the production server is not required.**
 
-Чтобы код блоков мог работать в приложении, блоки собирают в бандлы. Сборка бандла выполняется на основе декларации - специального файла с расширением `bemdecl.js`, где перечислены блоки, которые должны попасть в бандл. Пример декларации бандла:
+Blocks should be assembled into the bundles for usage in running application. Bundles formation is performed on the base of *bundle declaration*. It is a special file that contains the list of blocks which should be in the bundle. Bundle declaration example:
```

Comment:
-> for use in the running application. Bundle formation is based on the bundle declaration, which is a special file with the `bemdecl.js` extension that lists the blocks to bundle. Here is an example of a bundle declaration: